### PR TITLE
2017秋イベント対応(戦闘JSON変更) for E4

### PR DESCRIPTION
**Pull request by skywalker999u**

### CODE REVIEW

**Comment by hkuno9000** [comment]
@skywalker999u 
This pull-request #100 has conflicts. Please rebase or resolve

**Comment by skywalker999u** [comment]
@hkuno9000 it has been rebased, sorry about that

File: devtools.js [hunk 1]
```diff
@@ -2463,19 +2484,25 @@ function on_battle(json, battle_api_name) {
 	}
 	calc_damage(result, "先制対潜", d.api_opening_taisen,  f_nowhps, e_nowhps, f_nowhps_c, e_nowhps_c);	// 対潜先制爆雷攻撃.　2016-06-30メンテ明けから追加.
 	calc_damage(result, "開幕雷撃", d.api_opening_atack,   f_nowhps, e_nowhps, f_nowhps_c, e_nowhps_c);	// 開幕雷撃.
-	calc_damage(result, "夜戦砲撃", d.api_hougeki,         f_nowhps, e_nowhps, f_nowhps_c, e_nowhps_c);	// midnight
 	switch ($combined_flag) {
```
Comment (hkuno9000) [inline devtools.js]: why dit you remove the line?

File: devtools.js [hunk 2]
```diff
@@ -2463,19 +2484,25 @@ function on_battle(json, battle_api_name) {
 	}
 	calc_damage(result, "先制対潜", d.api_opening_taisen,  f_nowhps, e_nowhps, f_nowhps_c, e_nowhps_c);	// 対潜先制爆雷攻撃.　2016-06-30メンテ明けから追加.
 	calc_damage(result, "開幕雷撃", d.api_opening_atack,   f_nowhps, e_nowhps, f_nowhps_c, e_nowhps_c);	// 開幕雷撃.
-	calc_damage(result, "夜戦砲撃", d.api_hougeki,         f_nowhps, e_nowhps, f_nowhps_c, e_nowhps_c);	// midnight
```
Comment (hkuno9000) [inline devtools.js]: why this line was removed?

Comment (skywalker999u) [reply devtools.js]: It has been relocated before the new night battle api_n_hougehi1/2 lines (at line 2466) to make all night battle related codes go before all day battle codes

Comment (hkuno9000) [reply devtools.js]: i see.

**Pull request merged**